src/windshaft/windshaft-map.js

```diff
@@ -71,10 +71,12 @@ var WindshaftMap = Backbone.Model.extend({
 
         // TODO: Revisit this (will layerIndex work for NamedMaps??)
         // Should we move it somewhere else?
-        _.each(layers, function (layer, layerIndex) {
+        _.each(options.layers, function (layer, layerIndex) {
```

Comment:
this change provokes that torque layers doesn't receive the proper meta and torque widget is not working at all.